ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -11,7 +11,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "draftSDO",
     "UserRole": "caseworker-publiclaw-gatekeeper",
-    "CRUD": "R"
+    "CRUD": "CRU"
```

Comment:
remember to remove before merge

Reply:
Yeah, unless feature toggling is available by the time :)